src/Pages/NBALiveStats.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import '../App.css';
```

Comment:
should not be import styles from here. This is a component. Follow the same format as the others

Reply:
It's a page. Why would it be a component its a specific page with the landing page stuff